ui/pom.xml

```diff
@@ -27,11 +27,11 @@
       <plugin>
         <groupId>com.github.eirslett</groupId>
         <artifactId>frontend-maven-plugin</artifactId>
-        <version>1.9.1</version>
+        <version>1.12.1</version>
         <configuration>
           <workingDirectory>./</workingDirectory>
-          <nodeVersion>v16.13.0</nodeVersion>
-          <npmVersion>8.1.0</npmVersion>
+          <nodeVersion>v16.14.2</nodeVersion>
```

Comment:
We should re-evaluate if Node 16 LTS is the right tool.

Reply:
Do you mean about not update Node version or use a most recent version like Node.js 18?
According to this [link](https://angular.io/guide/versions) Angular 16 requires Node `^16.14.0 || ^18.10.0`

I tested with Node 18.10.0 and also is compatible to run tests, could be more adequate since Node 16.x.y is near to reach EOL.